src/DependencyInjection/InspectorExtension.php

```diff
@@ -36,7 +36,7 @@ public function load(array $configs, ContainerBuilder $container)
         $config = $this->processConfiguration($configuration, $configs);
         $container->setParameter('inspector.configuration.definition', $config);
 
-        if(true !== $config['enabled'] || empty($config['ingestion_key'])) {
+        if(empty($config['ingestion_key'])) {
```

Comment:
I agree with you. Furthermore I believe that this IF statement can be removed entirely.

Based on the code in the underlying library we can always load the service since the inspector service is able to enable or disable itself based on `ingetsion_key` existance or `enabled` flag state:

https://github.com/inspector-apm/inspector-php/blob/master/src/Configuration.php#L212

Reply:
Indeed! Removed the if entirely.